### PR TITLE
add css, html, images and .md files

### PR DESCRIPTION
**Pull request by kkasaija**
This is a first code review of my own repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# First-Review
```
Comment (kkasaija) [inline README.md:1]: Please follow the standard [template](https://github.com/microverseinc/readme-templatel) from Microvers

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">   
+        <link rel="stylesheet" href="styles.css">
+        <title>Embedding Images and Video</title>
+    </head>
+    <body>
+        <div class="header">
```
Comment (kkasaija) [inline index.html:11]: I advise you to use HTML semantic elements for better [SEO](https://moz.com/beginners-guide-to-seo)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                <div>
+                    <img class="thumbnail-1" src="thumbnail1.jpg" alt="video image">
+                </div>
+                <div>
+                    <img class="thumbnail-1" src="thumbnail2.jpg" alt="video image">
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (kkasaija) [inline index.html:61]: A new line at the end of the element always looks better. Its a good practice

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  display: flex;
+  margin-top: 20px;
+  margin-left: 250px;
+}
+.search-input{
+  display: block;
+  height: 35px;
+  width: 500px;
+  
+}
```
Comment (kkasaija) [inline styles.css:20]: The best practice and standard is to always have a space before and after the curly braces of the CSS selectors

**Review by kkasaija — COMMENTED** [review]
Thanks for the good work and effort put in to produce such a promising piece of code.
@Kasaija-Kenneth you are a champion in the making. just make changes to the highlighted areas.

**Review by leonardodiasb — APPROVED** [review]
Hi @Kasaija-Kenneth,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Good use of inline comments. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Using emojis/gifs in the review can make it more human-friendly. Remember that we must be polite and suggest changes instead of giving orders, and sometimes it is hard for the other person to understand our comments' tone.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**